test/core.jl

```diff
@@ -0,0 +1,35 @@
+#using Reproject: wcs_to_celestial_frame
+@testset "reproject-core" begin
+    if !isfile("gc_2mass_k.fits")
+        download("https://astropy.stsci.edu/data/galactic_center/gc_2mass_k.fits", "gc_2mass_k.fits")
+    end
+    if !isfile("gc_msx_e.fits")
+        download("https://astropy.stsci.edu/data/galactic_center/gc_msx_e.fits", "gc_msx_e.fits")
+    end
+    
+    imgin = FITS("gc_msx_e.fits")         # project this
+    imgout = FITS("gc_2mass_k.fits")        # into this coordinate
+    
+    hdu1 = astropy.io.fits.open("gc_2mass_k.fits")[1]
+    hdu2 = astropy.io.fits.open("gc_msx_e.fits")[1]
+    
+    function check_diff(arr1, arr2)
```

Comment:
I think the whole function is simply
```julia
all(isapprox(arr1, arr2))
```
or
```julia
all(isapprox(arr1, arr2, abstol = 1e-4))
```
if you want to specify a different absolute tolerance

Reply:
I tried using this function, in my version of reproject some corner cases arises where we get NaN but some value in python version. `check_diff` is written in that specific way to ignore those minor differences.

Reply:
I don't see how this function treats `NaN` specially:
```julia
julia> A = [NaN  2.4 5.9
            -3.6 NaN 0.18
            -1.7 6.0 2.3]
3×3 Array{Float64,2}:
 NaN      2.4  5.9 
  -3.6  NaN    0.18
  -1.7    6.0  2.3 

julia> check_diff(A, A)
false

julia> all(isapprox(A, A, nans = true))
true
```
`check_diff` wrongly reports that `A` and `A` are different when they're actually the same matrix.  `isapprox` with `nans=true` correctly says that they are approximately equal.

Reply:
check_diff transpose the second array as one comes from python. Thus check_diff(A,A) won't be true.